### PR TITLE
 #165273493 Users should be able to highlight and comment on any article

### PR DESCRIPTION
**Pull request by engjames**
**What does this PR do?**
 - Highlight and comment on any article.

**Description of the task to be completed?**
- Users should be able to highlight and comment on any article
image

**What are the relevant tasks?**
[#165273493](https://www.pivotaltracker.com/story/show/165273493)

### CODE REVIEW

File: authors/apps/comments/tests/base.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def setUp(self):
         )
         self.comment = {
             "comment": {
-                "body": "His name was my name too."
+                "body": "His name was my name too.",
```
Comment (mwinel) [inline authors/apps/comments/tests/base.py:76]: Avoid unnecessary changes.

**Review by mwinel — APPROVED** [review]

**Review by joelmugaya — APPROVED** [review]
LGTM

File: authors/apps/comments/views.py [hunk 2]
```diff
@@ -33,6 +33,23 @@ def post(self, request, **kwargs):
         article = self.get_article(kwargs['article_id'])
         comment = request.data.get('comment', {})
         comment['article'] = article.id
+        first_index = comment.get('first_index', 0)
+        last_index = comment.get('last_index', 0)
+        if "first_index" in comment and "last_index" in comment:
+            if not isinstance(first_index, int) or not isinstance(last_index, int):
```
Comment (archibishop) [inline authors/apps/comments/views.py:40]: @engjames Can this validation to check if these values provided(`validate_first_index, validate_last_index`) are ints be placed in the serialiser class.

**Review by archibishop — APPROVED** [review]
LGTM @engjames I am merging this.
